doc/plugins.md

```diff
@@ -8,10 +8,12 @@ This page lists existing plugins.
 
 ## Contents
 
-*   [List of plugins](#list-of-plugins)
-*   [List of utilities](#list-of-utilities)
-*   [Use plugins](#use-plugins)
-*   [Create plugins](#create-plugins)
+- [Plugins](#plugins)
+  - [Contents](#contents)
+  - [List of plugins](#list-of-plugins)
+  - [List of utilities](#list-of-utilities)
+  - [Use plugins](#use-plugins)
+  - [Create plugins](#create-plugins)
```

Comment:
Can you run `npm install` and `npm test`, that should reformat everything according to project style, or undo these changes manually? Thanks!

Reply:
done!